src/Assent/Reporters/DiffReporter.cs

```diff
@@ -49,7 +50,7 @@ public void Report(string receivedFile, string approvedFile)
                 if (program.Launch(receivedFile, approvedFile))
                     return;
 
-            throw new Exception("Could not find a diff program to use");
+            _consoleReporter.Report(receivedFile,approvedFile);
```

Comment:
Instead of adding it here, could you please add this to the end of the two arrays that set `DiffReporter.DefaultDiffPrograms`. This allows the end user to remove this reporter if they want.